src/shared/components/palette/fixed-list/write-fixed-list-field.component.ts

```diff
@@ -12,6 +12,6 @@ export class WriteFixedListFieldComponent extends AbstractFieldWriteComponent im
 
   ngOnInit() {
     let notEmpty = this.caseField.value !== null && this.caseField.value !== undefined;
-    this.fixedListControl = this.registerControl(new FormControl(notEmpty ? this.caseField.value : null));
+    this.fixedListControl = this.registerControl(new FormControl(notEmpty ? this.caseField.value : ''));
```

Comment:
worth adding a unit test for **notEmpty = false** for line `notEmpty ? this.caseField.value : ''`?